buildenv/jenkins/testJobTemplate

```diff
@@ -309,6 +309,11 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							stringParam('JDK_IMPL', JDK_IMPL_BASE, "JDK Base implementation, e.g. hotspot, openj9")
 							stringParam('CUSTOMIZED_SDK_URL', "", "Customized SDK url, need to set when SDK_RESOURCE=customized")
 							stringParam('CUSTOMIZED_SDK_URL_CREDENTIAL_ID', "", "Only use this if you are pulling an JDK from a site that needs credential")
+							stringParam('BUILD_LIST', "", "Specific test directories to compile. This can be a comma-separated list of directories. <br/>
```

Comment:
We already have that stringParam('BUILD_LIST', ...). So the line https://github.com/adoptium/aqa-tests/blob/master/buildenv/jenkins/testJobTemplate#L328 need to update.

Reply:
For this line, description part need to be updated as mentioned in #2934

Reply:
Done